frontend/src/components/CommentForm.tsx

```diff
@@ -35,4 +43,4 @@ class CreateCommentForm extends React.Component {
 
 const WrappedCommentForm = Form.create({ name: 'create_comment' })(CreateCommentForm);
 
-export default connect(null, null)(WrappedCommentForm);
\ No newline at end of file
+export default connect(null, null)(WrappedCommentForm);
```

Comment:
will you be adding logic here for state/props mapping to store?

Reply:
Probably not, I guess that `connect` is redundant?

Reply:
probably yup :)